doc/userguide/lua/libs/packetlib.rst

```diff
@@ -0,0 +1,164 @@
+Packet
+------
+
+Packets are exposed to Lua scripts with ``suricata.packet``
+library. For example::
+
+    local packet = require("suricata.packet")
+
+Initialization
+~~~~~~~~~~~~~~
+
+``get``
+^^^^^^^
+
+Init the packet for use in the script. The packet is the current packet the engine is processing.
+
+::
+
+    p = packet.get()
+
+
+Time
+~~~~
+
+``timestamp``
+^^^^^^^^^^^^^
+
+Get packet timestamp as 2 numbers: seconds & microseconds elapsed since
+1970-01-01 00:00:00 UTC.
+
+::
+
+    p = packet.get()
+    local sec, usec = p:timestamp()
+
+
+``timestring``
+^^^^^^^^^^^^^^
+
+Get packet timestamp as a string in the format: `11/24/2009-18:57:25.179869`
```

Comment:
Actually, to double back on this, wouldn't it make more sense to be the same format of EVE? `2025-02-10T20:56:43.996324+0000`?

Reply:
> `timestring_fastlog` - this code, it's used by fast log, alert debug log, http log, tls log, tls store, old lua
> `timestring_eve` - your suggestion

This looks good.  Then `timestring` when we do a standards compliant format, or `timestring_rfc3339` to be explicit.